Sources/MountebankSwift/Models/Stub/Response/Is/Is+initializers.swift

```diff
@@ -53,12 +53,13 @@ extension Is {
         statusCode: Int = 200,
         headers: [String: String]? = nil,
         body: any Codable,
+        encoder: JSONEncoder? = nil,
```

Comment:
I've chosen to inject the custom encoder here because this gives the best flexibility. The only downside is that users will have to pass it along every time they are using this... 

The alternative would be a global custom encoder, but I think that is less straightforward in terms of usage. Users will have to look longer where they will have to set it.